code/game/objects/items/tools/crowbar.dm

```diff
@@ -112,3 +112,33 @@
 	usesound = 'sound/items/jaws_pry.ogg'
 	force = 10
 	toolspeed = 0.5
+
+/obj/item/crowbar/combat
+	name = "combat crowbar"
+	desc = "Advanced crowbar 'borrowed' from combined alliance."

```

Comment:
desc = "**An a**dvanced crowbar 'borrowed' from **the** combined alliance."

Reply:
It's a meme description right now, but thanks.